docs/02-config.md

```diff
@@ -0,0 +1,24 @@
+# Configuring the plugin
+
+Phonehome exposes a single controller endpoint at `/actions/phonehome/api` with information about your Craft installation and the server environment it is installed in. 
+
+The plugin can be configured either via its settings page in the Craft control panel or via its config file, a starter version of which will be automatically created at `config/phonehome.php` as part of the plugin's install command.
+
+## Getting started
+
+The config file supports the following keys:
+
+- `token` - a shared, secret, token that will be used to authenticate requests to the Phone Home API. This should be set to a secure, random string.
```

Comment:
This token will be generated when the user adds the site to the Phone Home dashboard. It is not something customizable.